### PR TITLE
Pay 5565

### PR DESCRIPTION
**Pull request by anshikanigam15**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by larslunde-hmcts** [comment]
checking pipeline errors

**Comment by thomast1906** [comment]
As discussed with Anshika, missed patches for Demo.yaml
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/common-overlay/fees-pay/kustomization.yaml#L11

File: k8s/namespaces/fees-pay/ccpay-payment-api-int/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-api-int
+spec:
+  releaseName: ccpay-payment-api-int
+  values:
+    java:
+      image: hmctspublic.azurecr.io/payment/api:pr-1202-27bcf84-20220630101545 #{"$imagepolicy": "flux-system:demo-ccpay-payment-app"}
```
Comment (thomast1906) [inline k8s/namespaces/fees-pay/ccpay-payment-api-int/demo.yaml:9]: Current imagepolicy is not reflective for pr-1202

Comment (anshikanigam15) [reply k8s/namespaces/fees-pay/ccpay-payment-api-int/demo.yaml]: Updated image to 1190 which is currently deployed in demo

**Review by thomast1906 — APPROVED** [review]

File: k8s/namespaces/fees-pay/kustomization.yaml [hunk 2]
```diff
@@ -12,4 +12,7 @@ bases:
 - ccpay-bubble-frontend/ccpay-bubble-frontend.yaml
 - ccpay-payment-api/ccpay-payment-api.yaml
 - dead-letter-queue-process/dead-letter-queue-process.yaml
+- ccpay-payment-api-int/ccpay-payment-api-int.yaml
```
[the remark this answers is not in the document]
Comment (anshikanigam15) [reply k8s/namespaces/fees-pay/kustomization.yaml]: demo - removed from here , Thanks

**Review by anoojhmcts — APPROVED** [review]

File: k8s/namespaces/fees-pay/ccpay-refunds-api-int/demo.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-refunds-api-int
+  annotations:
```
[the remark this answers is not in the document]
Comment (anshikanigam15) [reply k8s/namespaces/fees-pay/ccpay-refunds-api-int/demo.yaml]: Removed refund update

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**